main-app/src/main/resources/conf/driverTemplate.yaml

```diff
@@ -9,4 +11,4 @@ metadata:
       uid: ${CONTROLLER_UID}
 spec:
   containers:
-    - name: spark-kubernetes-driver
+    - name: fabric-action
```

Comment:
If this name is being changed, will need to update it in the main-app/src/main/resources/python/submit_job.py to ensure it can find the right container for status watching. Also, I don't remember exactly, but I think there was an issue where that container was still going to the default name of spark-kubernetes-driver.

Reply:
```
Annotations:  kubectl.kubernetes.io/default-container: spark-kubernetes-driver                                                                                                                                                   │
│               kubectl.kubernetes.io/default-logs-container: spark-kubernetes-driver  
```
Should we use these annotations to pick container in pod? @mcriscolo-cs @jgielstra-cs

Reply:
Updated container name in submit_job